site/viamguides/src/sticker-wizard/sticker-wizard.md

```diff
@@ -0,0 +1,191 @@
+
+<div>
+  <img width="350" src="assets/intro.png">
+  <img width="350" src="assets/question.png">
+</div>
+
+We built our web application with [SvelteKit](https://svelte.dev/docs/kit/introduction) and [Threlte](https://threlte.xyz/). We used [Aseprite](https://www.aseprite.org/)-created pixel art assets with the [<AnimatedSpriteMaterial>](https://threlte.xyz/docs/reference/extras/animated-sprite-material) Threlte component to create the animations. Our web app has the following basic components:
+- Intro sequence
+- Quiz (Display questions, accept responses)
+- Results view (with dispense sticker button)
+
+<div>
+  <img width="500" src="assets/website-draft.png">
+</div>
+
+To authenticate your website with your machine, use Viam API keys. We recommend setting up an operator API key so users don't have write access to your machine (and use a `.env` file to avoid committing your API keys to your Github repo!)  
+
+See source code here (TODO)
+
+## Use Viam to run a local web server
+Through Viam, we can host our web server directly on the Raspberry Pi using a [process](https://docs.viam.com/configure/processes/). A process runs every time your machine starts up.
+1. Download your web app's source code to your Raspberry Pi
+    - We recommend uploading your source code to Github and then downloading `git` on your Raspberry Pi to pull changes from your website
+2. Follow instructions from your selected web framework to run your production bundle locally from your Pi. Make sure you can access it from your local network!
+    - In SvelteKit, you can run your [production bundle locally](https://svelte.dev/docs/kit/building-your-app#Preview-your-app) with `vite preview`
+    - Expose it on your [local network](https://www.ryanfiller.com/blog/tips/sveltekit-local-network) with the `--host` flag
+3. Create a bash script to run your production bundle locally. Call it `run-web-server.sh`.
+4. Add a process to your machine config
+    - Configure the executable to be the absolute path to `run-web-server.sh`
+    - Set the working directory to be the absolute path to the folder containing your bash script
+
+![web server config](assets/web-server-process.png)
+
+5. Test!
+    - Save your config
+    - Go to the logs tab, and check for logs verifying that your web server is up and running
+    - Verify that you can access your website on a separate device that is also on your local network
+
+To make your local web server available to those not on your local network, use a service like [ngrok](https://ngrok.com/). You can set up ngrok with Viam with a process in the same way so it is always available when your machine is running.
+
+## Finishing touches
+
+That's it! We added a few finishing touches to enhance the experience, but these are completely optional.
+
+![finishing touches](assets/finishes.jpeg)
+
+### LEDs
+  - Connect some LEDs to the SPI pin of the Raspberry Pi
```

Comment:
I like to use pinout.xyz to highlight certain pins or protocols when referencing them in tutorials, for example all SPI pins on a Raspberry Pi: https://pinout.xyz/pinout/spi

We should also be specific about which LEDs since there are many types and not all a driven by SPI. It looks like the sticker wizard is using WS2812 (?) / neopixel addressable RGB LEDs.

Reply:
updated! thanks for the link